core/cmd/remote_client_test.go

```diff
@@ -976,30 +1006,59 @@ func TestClient_P2P_DeleteKey(t *testing.T) {
 	err = app.Store.OCRKeyStore.UpsertEncryptedP2PKey(&encKey)
 	require.NoError(t, err)
 
-	keys, err := app.Store.OCRKeyStore.FindEncryptedP2PKeys()
-	require.NoError(t, err)
-	// Created  + fixture key
-	require.Len(t, keys, 2)
-
-	set := flag.NewFlagSet("test", 0)
-	set.String("file", "internal/fixtures/apicredentials", "")
-	c := cli.NewContext(nil, set, nil)
+	requireP2PKeyCount(t, app.Store, 2) // Created  + fixture key
 
-	err = client.RemoteLogin(c)
-	assert.NoError(t, err)
+	mustLogIn(t, client)
 
-	set = flag.NewFlagSet("test", 0)
+	set := flag.NewFlagSet("test", 0)
 	set.Bool("yes", true, "")
 	strID := strconv.FormatInt(int64(encKey.ID), 10)
 	set.Parse([]string{strID})
-	c = cli.NewContext(nil, set, nil)
+	c := cli.NewContext(nil, set, nil)
 	err = client.DeleteP2PKey(c)
 	require.NoError(t, err)
 
-	keys, err = app.Store.OCRKeyStore.FindEncryptedP2PKeys()
-	require.NoError(t, err)
-	// fixture key only
-	require.Len(t, keys, 1)
+	requireP2PKeyCount(t, app.Store, 1) // fixture key only
+}
+
+func TestClient_ImportExportP2PKeyBundle(t *testing.T) {
+	defer deleteKeyExportFile(t)
+
+	app, cleanup := cltest.NewApplication(t,
```

Comment:
Since we're moving to the new ETH mocking framework and away from this old one, would you mind replacing this initialization code with:
```go
	ethClient := new(mocks.Client)
    
	config, cleanup := cltest.NewConfig(t)
	defer cleanup()
	app, cleanup := cltest.NewApplicationWithConfig(t, config, ethClient)
	defer cleanup()
	verify := cltest.MockApplicationEthCalls(t, app, ethClient)
	defer verify()

	ethClient.On("BalanceAt", mock.Anything, mock.Anything, mock.Anything).Return(big.NewInt(42), nil)
	ethClient.On("GetLINKBalance", mock.Anything, mock.Anything, mock.Anything).Return(assets.NewLink(42), nil)

```
I believe that's all you should need... should "just work"